modules/ROOT/pages/managing-applications-on-your-own-servers.adoc

```diff
@@ -78,6 +79,14 @@ image::managing-applications-on-your-own-servers-96463.png[]
 [NOTE]
 Note that the server must be running for this option to be available. In case of a Cluster or Server Group, at least one of its nodes must be running.
 
+== Download an Application Archive
+
```

Comment:
Check wording consistency: in one occurrence, you say "application archive", but in another,  you say "app archive." The title has "Application Archive." Also, should the application *Settings* be app *Settings*?